Makefile

```diff
@@ -26,22 +26,23 @@ extra_quality_checks: deps_table_update
 	python utils/check_table.py
 	python utils/check_dummies.py
 	python utils/check_repo.py
-	python utils/style_doc.py src/transformers docs/source --max_len 119
+	python utils/style_doc.py src/transformers docs/source --max_len 119 --check_only
```

Comment:
If I'm not mistaken, this prevents your `style_doc.py` to do changes when used with `make fixup`, is that voluntary?